test/test_table.py

```diff
@@ -0,0 +1,147 @@
+from hpack.table import HeaderTable
+import pytest
+import sys
+_ver = sys.version_info
+is_py2 = _ver[0] == 2
+is_py3 = _ver[0] == 3
+
+class TestHeaderTable(object):
+    def test_getitem_dynamic_table(self):
+        tbl = HeaderTable()
+        off = len(HeaderTable.STATIC_TABLE)
+        val = (b'TestName', b'TestValue')
+        tbl.add(val[0], val[1])
```

Comment:
A general note for the tests: brevity is not particularly valuable in tests, so feel free to use longer variable names. The tests are part of the product, after all.